PTA/pta.h

```diff
@@ -16,10 +16,11 @@ QT_FORWARD_DECLARE_CLASS(ItemAPI)
 
 struct PItem;
 
-enum price_check_flag : uint32_t
+enum search_check_flag : uint32_t
 {
     PC_SIMPLE = 0,
     PC_ADVANCED,
+	WIKI_SEARCH,
```

Comment:
Please make sure that your editor is set to insert spaces and not tab characters for tabs.